src/Apps/Partner/Components/PartnerArtists/PartnerArtistsCarousel/PartnerArtistsCarousel.tsx

```diff
@@ -63,10 +65,11 @@ export const PartnerArtistsCarouselFragmentContainer = createFragmentContainer(
         ) {
           edges {
             node {
-              ...CellArtist_artist
+              id
```

Comment:
Is this being used? Note that `id` = relay globally unique ID and has no relationship to our API data. (id = internalID in terms of data)

Reply:
good point I think you're right and its not necessary.